.github/workflows/semantic-pr.yml

```diff
@@ -0,0 +1,20 @@
+name: semantic-pull-request
```

Comment:
Let's take this chance to rename these workflows to something more "user-friendly".  Maybe like `Verify Semantic Pull Request`.